### ISSUE TITLE
TE v2 - Change wording on Presentation Page

### ISSUE THREAD

**Issue by misulicus** [wp-e-commerce/WP-e-Commerce#1983]
http://www.localhost.com/wpec-master/wp-admin/options-general.php?page=wpsc-settings&tab=presentation

Remove beta wording and maybe rephrase the top text

**Comment by danmilward** [comment]
AND move the Styling Settings (which are confusing and meaningless to all human beings that don't know what that means) to below the main settings. They should be underneath the setting that I can decipher the meaning of.

I say this as a WordPress user since it was B2, I am the lowest type of user, we must make confusing UI less confusing. Or lets just release 4.0 and 3.9.some-other-number.

**Comment by JustinSainton** [comment]
I'm all for revamping the wording - PRs, please!  Or at least suggestions :)

**Comment by danmilward** [comment]
Also we should include Fancy Notifications - some of our longest standing committers have said they want it and use it on their client sites aka @benhuson @leewillis77 :)

**Comment by danmilward** [comment]
Dunno on what planet removing the things that make us awesome is awesome. Take away the awesome, puts more focus on what is not awesome :(

**Comment by JustinSainton** [comment]
@danmilward I'm not sure I really understand what you're saying.  I assume you're referencing #516. There was a lot of back-and-forth on that, _years ago_.

I summarized the point made there and [laid out a proposal about two months ago](https://github.com/wp-e-commerce/WP-e-Commerce/issues/516#issuecomment-117918182). Your feedback on that proposal would be welcomed.

I like and use Fancy Notifications (and AJAX) for shopping carts, FWIW.  That doesn't mean 4.0 needs to ship with solutions for those in core.

**Comment by benhuson** [comment]
I'm not averse to to something like Fancy Notifications no longer being in core as look as there is a plugin/add-on in existence that will replace it so users can upgrade without losing functionality - that would be a really bad experience for existing users.

I know I was tagged on that ticket to maybe look add some sort of AJAX functionality that would allow for Fancy Notifications as a plugin but I have not had time available to look into that as yet.

**Comment by danmilward** [comment]
Emphasis on

" bad experience for users "

Agreed, and we need a Plugin linked in the marketplace as an interim while no themes will have this.

Ben, I don't suppose you could make the Plugin?

d

**Comment by JustinSainton** [comment]
Would _love_ to have this plugin built.  Even cues in core for it would be great.  Something like a notification template part that could be hooked into, a jQuery event listener that could be unbound by default, and have other methods bound to it from themes/plugins - I think that would be awesome.

**Comment by benhuson** [comment]
@JustinSainton Yep, that's the sort of thing I was thinking. I haven't had time to look into it though. In fact I need to find time to have a good work through the v2 theme engine which I've not yet familiarised myself with in depth :fearful:

**Comment by JustinSainton** [comment]
I believe we're all settled here. Nearly all major presentational settings have been moved to the Customizer, and will continue to be moved there.
